apps/dapp-console-api/src/testhelpers/session.ts

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
this prevents the following type error:
<img width="640" alt="Screenshot 2024-03-28 at 3 51 15 PM" src="https://github.com/ethereum-optimism/ecosystem/assets/15234569/bdc779ea-6663-4ef4-9b41-ec2415ee0289">